### PR TITLE
[Docs] Readme update to describe purpose of this repo

### LINKED ISSUE #2: [Docs] Update README to describe project
---
**Contributor Resources**
- [Meshery documentation site](https://docs.meshery.io/)
- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)

### PR DESCRIPTION
**Pull request by leecalcote**
**Description**

This PR fixes #2 and better captures the purpose of this repo.

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
The failed build checks are expected.

File: README.md [hunk 1]
```diff
@@ -1,37 +1,64 @@
...
+</div>
 
 <p style="clear:both;">
-<a href ="https://layer5.io/community/meshmates"><img alt="MeshMates" src=".github/readme/images/Layer5-MeshMentors-1.png" style="margin-right:10px; margin-bottom:15px;" width="28%" align="left"/></a>
-<h3>Find your MeshMate</h3>
+<h2><a href="https://layer5.io/meshery">Meshery</a></h2>
+<a href="https://meshery.io"><img src="img/readme/meshery-logo-light-text-tag.svg"
+style="margin:10px;" width="125px" 
+alt="Meshery - the Collaborative Cloud Manager" align="left" /></a>
+As a self-service engineering platform, <a href="https://meshery.io">Meshery</a> enables collaborative design and operation of cloud native infrastructure. Through it's extension points, Meshery offers the ability to optionally plugin adapters in order to more deeply integrate with specific systems like <a href="https://getnighthawk.io">Nighthawk</a>.
```
Comment (gyohuangxin) [inline README.md:32]: https://getnighthawk.io can't be accessed, it should be https://getnighthawk.dev.

File: README.md [hunk 2]
```diff
@@ -1,37 +1,64 @@
...
 
-<p>MeshMates are experienced Layer5 community members, who will help you learn your way around, discover live projects and expand your community network. 
-Become a <b>Meshtee</b> today!</p>
-
-Find out more on the <a href="https://layer5.io/community">Layer5 community</a>. <br />
-<br /><br /><br /><br />
+<h2><a href="https://layer5.io/meshery">Nighthawk</a></h2>
+<a href="https://getnighthawk.io"><img src="img/readme/nighthawk-full.svg"
```
Comment (gyohuangxin) [inline README.md:35]:
```suggestion
<a href="https://getnighthawk.dev"><img src="img/readme/nighthawk-full.svg"
```

File: README.md (hunk 1 above)
Comment (gyohuangxin) [reply README.md:32]:
```suggestion
As a self-service engineering platform, <a href="https://meshery.io">Meshery</a> enables collaborative design and operation of cloud native infrastructure. Through it's extension points, Meshery offers the ability to optionally plugin adapters in order to more deeply integrate with specific systems like <a href="https://getnighthawk.dev">Nighthawk</a>.
```

**Review by gyohuangxin — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (leecalcote) [reply README.md:32]: Oh! Thank you, @gyohuangxin 👍

File: README.md (hunk 2 above)
Comment (leecalcote) [reply README.md:35]: Accepting suggestion...

**Review by gyohuangxin — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**